browser/cypress/e2e/home_page.cy.ts

```diff
@@ -0,0 +1,108 @@
+// Sample test for home page
+
+describe('Home Page', () => {
+  // make sure basics of the page succesfully load
+  it('Home page loads successfully', () => {
+    cy.visit('/')
+
+    // Make sure the page has the landing splash
+    cy.contains('Genome Aggregation Database')
+    cy.contains('gnomAD browser')
+
+    // Make sure page content headers are there
+    cy.contains('New to gnomAD?')
+    cy.contains('About')
```

Comment:
Looks good, but perhaps we could interact with the version selector here? Consider that that's going to change soon, inasmuch as v4 will shortly be an option.